packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.3.1
```

Comment:
You've updated the version and changelog in the wrong package; the change is to `google_sign_in_android`.